proto/raft_serverpb.proto

```diff
@@ -252,6 +252,8 @@ enum ExtraMessageType {
     MsgGcPeerResponse = 12;
     MsgFlushMemtable = 13;
     MsgRefreshBuckets = 14;
+    MsgAckCommittedIndexRequest = 15;
```

Comment:
Ack is more like that someone acknowledgment something. May rename to Fetch/Ask or something?